docker-compose/baker+wallet-proxy+identity-provider.yaml

```diff
@@ -0,0 +1,168 @@
+    - postgresql
+    entrypoint: ["/concordium-node",
+       "--no-dnssec",
+       "--no-bootstrap",
+       "--debug",
+       "--config-dir", "/var/lib/concordium",
+       "--data-dir", "/var/lib/concordium",
+       "--baker-credentials-file", "/var/lib/concordium/baker-0-credentials.json",
+       "--bootstrap-node", "bootstrapper:8888",
+       "--transaction-outcome-logging-database-host", "postgresql",
+       "--transaction-outcome-logging-database-port", "5432",
+       "--transaction-outcome-logging-database-name", "concordium",
+       "--transaction-outcome-logging-database-username", "concordium",
+       "--transaction-outcome-logging-database-password", "concordium",
+       "--rpc-server-addr", "0.0.0.0",
+       "--desired-nodes", "0"
+       ]
+    environment:
+    - RUST_BACKTRACE=1
+    - RUST_LOG=info
+    ports:
+    - 10000:10000
+
+  # The collector that posts data to the collector backend to display it.
+  collector:
+    image: concordium/dev-node:latest
+    depends_on:
+    - baker_reporter
+    environment:
+    - RUST_BACKTRACE=1
+    - RUST_LOG=info
+    entrypoint: ["/node-collector",
+      "--collector-url", "http://collector_backend:10000/nodes/post",
+      "--grpc-host", "http://baker_reporter:10000",
+      "--node-name", "reporter-node"
+    ]
+
+  # database where the transaction log is stored for the wallet-proxy.
+  # The database is persisted to `db-volume`
+  postgresql:
+    image: postgres:latest
+    # persist the database through restarts
+    volumes:
+      - db-volume:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: concordium
+      POSTGRES_USER: concordium
+      POSTGRES_DB: concordium
+    ports:
+    - 5432:5432
+
+  wallet_proxy:
+    image: abizjak/wallet-proxy:latest
+    depends_on:
+    - baker_reporter
+    - postgresql
+    environment:
+    - DB_SLEEP=5 # delay start for 5 seconds to wait for the postgres db to start
+    - WALLET_PROXY_GRPC_IP=baker_reporter
+    - WALLET_PROXY_GRPC_PORT=10000
+    - WALLET_PROXY_DATABASE="host=postgresql port=5432 user=concordium dbname=concordium password=concordium"
+    # comment the following line to disable gtu-drop
+    # NB: paths are in the containers.
+    - WALLET_PROXY_ACCOUNT_FILE=/data/gtu-drop-account.json
+    - WALLET_SERVER_INFOS_FILE=/data/ips-with-metadata.json
+    # mount the genesis data directory into the container to get the following
+    # files
+    # - data/gtu-drop-account.json
+    # - data/ips-with-metadata.json
+    # The mounted directory must contain these two. If this is not the case the
+    # values of environment variables WALLET_PROXY_ACCOUNT_FILE and
+    # WALLET_SERVER_INFOS_FILE must be changed accordingly
+    volumes:
+      # path to genesis data, relative to this file, or absolute path
+      - $HOST_ROOT/configuration/wallet-proxy:/data
+    entrypoint:
+    - /start.sh
+    ports:
+    - 14000:3000
+
+  identity_verifier:
+    image: abizjak/test-identity-provider:latest
```

Comment:
This is currently using my own space until images are posted publicly under concordium.